scripts/machine-launch-script.sh

```diff
@@ -343,7 +344,9 @@ set -o pipefail
 
 
     argcomplete_extra_args=()
-    if [[ "$INSTALL_TYPE" != system ]]; then
+    if [[ "$INSTALL_TYPE" == system ]]; then
+	argcomplete_extra_args=( --dest "${CONDA_ENV_BIN}/../share/bash-completion/completions" )
```

Comment:
I looked more into how things work with the yum package and installing the argcomplete global completion script.  It looks like the scripts in `/etc/bash_completion.d/` are different from `/usr/share/bash-completion/completions`. I'm still digging into what ends up sourcing the `/etc/bash_completion.d/*` scripts with the yum package and how that isn't being done with the conda package.